demos/shared-ros/ros-app/bin/run-system

```diff
@@ -21,4 +26,4 @@ export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$ROS_BASE/lib/$TRIPLET
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$ROS_BASE/usr/lib
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$ROS_BASE/usr/lib/$TRIPLET
 
-roslaunch listener talk_and_listen.launch
+roslaunch listener talk_and_listen.launch "$@"
```

Comment:
this should ideally have an `exec` but given the urgency of this PR, make it in a new one ;-)